spec.bs

```diff
@@ -934,6 +983,30 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
+1. If |config|["{{AuctionAdConfig/auctionNonce}}"] [=map/exists=]:
+  1. Set |auctionConfig|'s [=auction config/auction nonce=] to the result of running
+    [=get uuid from string=] with |config|["{{AuctionAdConfig/auctionNonce}}"].
+1. If |config|["{{AuctionAdConfig/additionalBids}}"] [=map/exists=]:
+  1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:
+    * |config|["{{AuctionAdConfig/auctionNonce}}"] does not [=map/exist=];
+    * |config|["{{AuctionAdConfig/interestGroupBuyers}}"] does not [=map/exist=], or it's [=list/empty=];

```

Comment:
Just to be clear, we do want to throw an error if this IS empty? That means we require it to be non-empty in this case (just making sure we don't have an extra negative here)?